repo/packages/K/koding/0/config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "type": "object",
+  "properties": {
+    "networking": {
```

Comment:
add name (name of the service instance), cpu (CPU share to allocate) and memory (memory to allocate) properties. Refer other packaged for this.